Jenkinsfile

```diff
@@ -131,6 +131,9 @@ pipeline {
                     },
                     "Audit - ${env.STAGE_NAME}": {
                       sh 'ci/test rspec_audit'
+                    },
+                    "Policy Parser - ${env.STAGE_NAME}": {
+                      sh 'cd gems/policy-parser && ./test.sh'
```

Comment:
Just noting that whichever one of this or @jonahx's [CI cleanup PR](https://github.com/cyberark/conjur/pull/1913) lands second will need to update this to conform to the new testing standards. My preference would be for Jonah's to merge first and we update the test entrypoint in this PR, but I don't know how close that one is.

Reply:
Agreed.  We should get Jonah's PR in first.  I'll adjust after that.